control-plane/config/post-install/200-controller-cluster-role.yaml

```
@@ -20,4 +20,19 @@ metadata:
   name: knative-kafka-controller-post-install
   labels:
     app.kubernetes.io/version: devel
-rules: []
+rules:
+  # we need to be able to delete old deployments
+  - apiGroups:
+      - "apps"
+    resources:
+      - "deployments"
+    verbs:
+      - "delete"
```

```suggestion
      - "delete"
      - "list"
```
I think we need this to be able to make our list call in the post install code